AuthenticationTests/LogIn/LoginViewModelTests.swift

```diff
@@ -0,0 +1,298 @@
+//
+//  LoginViewModelTests.swift
+//  AuthenticationTests
+//
+//  Created by Guido Marucci Blas on 3/7/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Quick
+import Nimble
+@testable import Authentication
+
+import ReactiveCocoa
+import enum Result.NoError
+
+
+class LoginViewModelSpec: QuickSpec {
```

Comment:
As a general rule refactor context to start with `context("when ...")` and avoid using `should` inside `it`